screens/CheckoutScreen.js

```diff
@@ -181,61 +184,113 @@ export default class CheckoutScreen extends React.Component {
     }
   };
 
+  // Returns index of the first product with a name starting with the given letter in products list.
+  // If no product starting with that letter exists, find the next product.
+  getIndexOfFirstProductAtLetter = letter => {
+    let prodList = this.state.products.filter(product => product.name.charAt(0) === letter);
+    let nextLetter = letter;
+    while (prodList.length === 0) {
+      nextLetter = String.fromCharCode(nextLetter.charCodeAt(0) + 1);
+      prodList = this.state.products.filter(product => product.name.charAt(0) === nextLetter);
+    }
+    return this.state.products.indexOf(prodList[0]);
+  };
+
+  // Takes in strings tab label (i.e. "A-K") and starting letter (i.e. "A") and returns a
+  // tab for the bottom alphabetical scroll bar.
+  alphabeticalScrollTab = (label, letter) => {
+    return (
+      <TabContainer
+        onPress={() =>
+          this._scrollView.scrollTo({
+            y: Math.floor(this.getIndexOfFirstProductAtLetter(letter) / 5) * 160
```

Comment:
ah, I can see why making this responsive would be very rough 😂

Reply:
I'm curious if this math will still work once the product list shrinks due to being store-specific?

Reply:
@anniero98 I talked to @wangannie about this and I think we ended up deciding that clerks would always see all products on the checkout screen? (I think it was because maybe they don't update the products available at a certain store in the Airtable before giving it to them physically but correct me if I'm wrong)

Reply:
Yeah I don't think it's going to be store specific